dev-db/percona-server/percona-server-8.0.26.16-r2.ebuild

```diff
@@ -721,7 +721,7 @@ pkg_config() {
 		local n_X
 		let n_X=${#template}-${#template_wo_X}
 		if [[ ${n_X} -lt 3 ]] ; then
-			echo "${FUNCNAME[0]}: too few X's in template ‘${template}’" >&2
+			echo "${FUNCNAME[0]}: too few X's in template '${template}'" >&2
```

Comment:
This one can move up to the previous commit maybe. It's part of user messaging even though not DESCRIPTION specifically.